### DISCUSSION TITLE
Duration for rotationAngle.

### DISCUSSION THREAD

**Discussion by blogwebpl** [alexandra-c/leaflet-tracking-marker#31] [General]
Hi.
It is possible to add duration to rotationAngle?

For example:

First position: rotationAngle = 0, duration = 1000
Second position: rotationAngle = 180

Angle changes in 1000ms from 0 to 180. (0 = 0, 500ms = 90, 750ms = 135 etc.)

best
Tomek

**Comment by alexandra-c** [comment]
Hello @blogwebpl ! Unfortunately right now the library does not handle rotation duration. If you wish you can contribute with a pull request to include this feature. 😊
